grammars/tree-sitter-python.cson

```diff
@@ -126,6 +128,8 @@ scopes:
   '"if"': 'keyword.control'
   '"else"': 'keyword.control'
   '"elif"': 'keyword.control'
+  '"match"': 'keyword.control'
+  '"case"': 'keyword.control'
```

Comment:
Do these work when Tree-sitter is enabled? I would think that the Tree-sitter parser would need to be upgraded to have the `match` and `case` keywords parsed.